src/db.rs

```diff
@@ -800,6 +862,57 @@ mod tests {
         }
     }
 
+    #[tokio::test]
+    async fn test_write_batch() {
```

Comment:
`wal_disable` portion does not seem to be tested.

Reply:
Fixed here: [e19a784](https://github.com/slatedb/slatedb/pull/264/commits/e19a7842283ef666991ef853838a47bfe22a29cd)